lib/utils/import/test.js

```diff
@@ -0,0 +1,14 @@
+import importNotes from './';
+
+describe('Importer', () => {
+  it('should throw when no notes are passed', () => {
+    expect(importNotes.bind(importNotes, null)).toThrow(/No notes to import./);
+  });
+
+  it('should throw when invalid json is passed', () => {
+    const bogusNotes = { actveNotes: [] };
+    expect(importNotes.bind(importNotes, bogusNotes)).toThrow(
```

Comment:
Out of curiosity, are these `bind` statements intended to do anything more than a simple `expect(() => importNotes(null))`? It had me wondering why a bind was necessary here.

Reply:
I think it was executing the function during the test instead of evaluating it for the result. Hope that makes sense :)

Reply:
Ok, if it's just to avoid the immediate call, then simply wrapping the call in a function (e.g. `expect(() => importNotes(null))` ) should be sufficient.

Reply:
Nice! Thanks.